src/classes/STG_InstallScript_TEST.cls

```diff
@@ -170,9 +170,12 @@ public with sharing class STG_InstallScript_TEST {
         
         //Verify that all the TDTM classes have been deactivated (since all the NPSP disable flags were on)
         List<Trigger_Handler__c> afterScriptConfig = [select Active__c, Class__c from Trigger_Handler__c where Class__c IN :tdtmClasses];     
-        //The -1 is because there are two NPSP flags (npe01__DISABLE_OpportunityAccount_trigger__c and 
-        //npe01__DISABLE_OpportunityContactRoles_trigger__c) that point to the same TDTM class (OPP_OpportunityContactRoles_TDTM)
-        System.assertEquals(tdtmClassesWithValues.size() - 1, afterScriptConfig.size());     
+        // This used to -1 because there are two NPSP flags (npe01__DISABLE_OpportunityAccount_trigger__c and
+        // npe01__DISABLE_OpportunityContactRoles_trigger__c) that point to the same TDTM class
+        // (OPP_OpportunityContactRoles_TDTM. However, a new trigger on the Payment object was added using the
+        // same PMT_Payment_TDTM class referenced in the above list of getNpspToCumulusMap(). That nets out the
+        // -1 to 0.
+        System.assertEquals(tdtmClassesWithValues.size(), afterScriptConfig.size());
```

Comment:
There was a comment here already that noted that the number of returned records should be `tdtmClassesWithValues.size()-1`. The new trigger handler record added in this story changed. I wanted to explain why the it looks the way it does now, without losing the historical context. It's possible that at `-1` might be needed again at some point if another trigger handler is added that changes the return count. If that happens, I thought the extra comment might help the dev know that adding `-1` would solve the issue.

Reply:
I am wondering if using the `Set` to store unique classes will remove the need for the comment and replace the code lines 162-168 altogether?

    Set<String> uniqueTdtmClasses = new Set<String>(
        TDTM_DefaultConfig.getNpspToCumulusMap().values()
    );

Reply:
Possibly, though we'd need to do the same with the list of `TriggerHandler__c` records returned since those could also include more than one record per Class Name. Would definitely make for a cleaner test.